src/vault/LiteVault.sol

```diff
@@ -84,6 +98,7 @@ contract LiteVault is IVault, ReentrancyGuard {
         if (token == address(0)) {
             payable(msg.sender).transfer(amount);
         } else {
+            // TODO: (RESTR) the same for early ERC20 tokens.
             require(IERC20(token).transfer(msg.sender, amount), "Transfer failed");
         }
```

Comment:
Suggestion: add bulk balanceOf() method for an array of user addresses to decrease the number of rpc calls

Reply:
Does it make sense to have methods for both single-user-multi-tokens and multi-users-single-token balance retrievals?